nusight2/src/client/components/navigation/view.tsx

```diff
@@ -27,7 +27,7 @@ const NavigationItemView = ({ url, Icon, children }: NavigationItemViewProps) =>
 );
 
 export const NavigationView = ({ nav }: { nav: NavigationConfiguration }) => (
-  <div className="dark bg-gray-800 dark:bg-auto-surface-1 border-r border-auto text-center flex flex-col justify-between items-center">
+  <div className="dark bg-gray-800 dark:bg-auto-surface-1 border-r border-auto text-center flex flex-col justify-between items-center h-screen overflow-y-auto">
```

Comment:
```suggestion
  <div className="dark bg-gray-800 dark:bg-auto-surface-1 border-r border-auto text-center flex flex-col justify-between items-center h-screen overflow-y-auto sticky top-0">
```

I think something like this would be good, not sure if we would ever want the sidebar scrolling out of view

Reply:
Thanks for the feedback. I’ll look into the navbar scrolling issue you mentioned. Regarding the kinematics tab, I’ve been working on it in a separate branch, so I’ll modify this PR to focus solely on the navbar changes.

Reply:
I’ve implemented your suggestion by adding `sticky top-0` to ensure the navbar remains visible while still allowing independent scrolling for its content. Let me know if there’s anything else I should adjust.